lib/src/apivideo_player_overlay.dart

```diff
@@ -311,37 +360,45 @@ class _ApiVideoPlayerOverlayState extends State<ApiVideoPlayerOverlay>
       onPressed: () {
         replay();
       },
-      iconSize: 60,
+      iconSize: _getPrimaryIconsSize(),
       icon: Icon(
         ApiVideoIcons.replayPrimary,
         color: widget.theme.controlsColor,
       ));
 
   Widget buildSlider() => Container(
         height: 60,
-        padding: const EdgeInsets.only(right: 5),
+        padding: const EdgeInsets.only(right: 1),
         child: Row(
           children: [
             Expanded(
-              child: Slider(
-                value: max(
-                    0,
-                    min(
-                      _currentTime.inMilliseconds,
-                      _duration.inMilliseconds,
-                    )).toDouble(),
-                // Ensure that the slider doesn't go over the duration or under 0.0
-                min: 0.0,
-                max: _duration.inMilliseconds.toDouble(),
-                activeColor: widget.theme.activeTimeSliderColor,
-                inactiveColor: widget.theme.inactiveTimeSliderColor,
-                onChanged: (value) {
-                  setCurrentTime(Duration(milliseconds: value.toInt()));
-                },
+              child: SliderTheme(
+                data: SliderThemeData(
+                  thumbShape: RoundSliderThumbShape(
+                    enabledThumbRadius: _getSliderRoundedIconSize(),
+                  ),
+                ),
+                child: Slider(
+                  value: max(
+                      0,
+                      min(
+                        _currentTime.inMilliseconds,
+                        _duration.inMilliseconds,
+                      )).toDouble(),
+                  // Ensure that the slider doesn't go over the duration or under 0.0
+                  min: 0.0,
+                  max: _duration.inMilliseconds.toDouble(),
+                  activeColor: widget.theme.activeTimeSliderColor,
+                  inactiveColor: widget.theme.inactiveTimeSliderColor,
+                  onChanged: (value) {
+                    setCurrentTime(Duration(milliseconds: value.toInt()));
+                  },
+                ),
               ),
             ),
             Text((_duration - _currentTime).toPlayerString(),
-                style: const TextStyle(color: Colors.white)),
+                style: TextStyle(
+                    color: Colors.white, fontSize: _getSliderTimerSize())),
```

Comment:
Ok, this is not an issue of this PIR but shouldn't it be `widget.theme` instead of `Colors.white`?